lua/terrortown/menus/gamemode/administration/entspawn.lua

```diff
@@ -0,0 +1,165 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 97
+CLGAMEMODESUBMENU.title = "submenu_administration_entspawn_title"
+
+local updateButtons = {}
+local updateCheckBoxes = {}
+local updateHelpBox = nil
+
+local function UpdateButtons(state)
+	state = state and not tobool(ttt2net.Get({"entspawnscript", "settings", "blacklisted"}))
```

Comment:
it can be called from two different sources. In one of them the state is not known. I could probably move this check to there. Hmm.

Edit: Moved it.